core/src/main/java/hudson/model/Run.java

```diff
@@ -1394,6 +1394,10 @@ public String getLength() {
         }
         
         public long getFileSize(){
+            if (length.length() == 0) {
```

Comment:
`StringUtils.isBlank()` ? Maybe it is an edge case, but the existing logic is not specific about what can be passed to this field (null, bogus blank strings, etc.). So being more protective would not hurt